src/server/pegasus_write_service_impl.h

```diff
@@ -675,7 +664,7 @@ class pegasus_write_service::impl : public dsn::replication::replica_base
         return status.code();
     }
 
-    // The resulted `expire_ts` is -1 if record is expired.
+    /// \returns 0 if success.
```

Comment:
I think the comment of this function should be more detailed, such as the expired case, not_found case, these case will also return 0, but in my view, they are not simply success.